### PR TITLE
Fs exercises view

### PR DESCRIPTION
**Pull request by fszwedo**
Exercises View created, to test please run ```npm run start:views``` and check the exercises tab (you might need to have a logged in user with apikey and api id)

### CODE REVIEW

**Review by robert-czwartosz — APPROVED** [review]
Looks well!

**Review by skorpiom — COMMENTED** [review]
Looks  great, but just one thing - I found that clicking cancel button takes user to the overview view.

**Comment by robert-czwartosz** [comment]
@skorpiom Bug you mentioned is in addFood and addExercise tiles.
Taking user to overview is caused by refreshing page after click Cancel button

**Comment by kjreczek** [comment]
I fixed this issue with cancel button in my component my diary food, feel free to copy and paste ([commit](https://github.com/KrystianKjjk/CodersCamp2020.Project.TypeScript.FitNotFat/pull/49/commits/a287daeb9f2ed77fdaf092528ad38ba20644c1c2))

File: Src/UIComponents/Overview/Overview.ts [hunk 1]
```diff
@@ -56,11 +56,11 @@ export function generateGaugesContent(User: User){
     //generate tile content
     //if user is heavier than goal
     if (User.goalWeight <= User.weights[0].weight){
-        generateGoalTile("Weight Goal", "overviewGoalWeightTile", User.weights[0].weight, User.goalWeight, userMaxWeight, "kg", true);
+        generateGoalTile("Weight Goal", "overviewGoalWeightTile", User.weights[0]?.weight, User.goalWeight, userMaxWeight, "kg", true);
     } 
     //if user is lighter than goal
     else{
-        generateGoalTile("Weight Goal", "overviewGoalWeightTile", User.weights[0].weight,  userMinWeight, User.goalWeight,  "kg", false);
+        generateGoalTile("Weight Goal", "overviewGoalWeightTile", User.weights[0]?.weight,  userMinWeight, User.goalWeight,  "kg", false);
```
[the remark this answers is not in the document]
Comment (fszwedo) [reply Src/UIComponents/Overview/Overview.ts:63]: true, I even did it this way in some other places, not sure why I did it like that here :) thanks for spotting that

File: Src/UIComponents/Overview/Overview.ts [hunk 2]
```diff
@@ -56,11 +56,11 @@ export function generateGaugesContent(User: User){
     //generate tile content
     //if user is heavier than goal
     if (User.goalWeight <= User.weights[0].weight){
-        generateGoalTile("Weight Goal", "overviewGoalWeightTile", User.weights[0].weight, User.goalWeight, userMaxWeight, "kg", true);
+        generateGoalTile("Weight Goal", "overviewGoalWeightTile", User.weights[0]?.weight, User.goalWeight, userMaxWeight, "kg", true);
```
[the remark this answers is not in the document]
Comment (fszwedo) [reply Src/UIComponents/Overview/Overview.ts:59]: done

**Comment by fszwedo** [comment]
applied a fix from Konrad, this bug should no longer occur @skorpiom

**Comment by skorpiom** [comment]
Works great!

**Review by skorpiom — APPROVED** [review]

**Pull request merged**